inputremapper/gui/user_interface.py

```diff
@@ -465,6 +472,11 @@ def show_status(self, context_id, message, tooltip=None):
             status_bar.push(context_id, message)
             status_bar.set_tooltip_text(tooltip)
 
+    @debounce
+    def check_on_typing(self, *_):
```

Comment:
Try to add a docstring. It should have the format of

```py
    def check_on_typing(self, *_):
        """Do this and that."""
```

The line with the comment should not be longer than 88 characters. A short sentence that fits in a single line.

Docstrings are super important. They will save you once you have to look at old code and don't know anymore what it is doing. Having to write a very short docstring for each function makes you think twice if that function is really doing something that can easily be explained. If it is hard to explain what a function is doing, it might need some redesigning.